mvi/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "template.mvi"
+    compileSdk = 33
```

Comment:
Add this to gradle version catalog, so it can easily be shared across modules in this project.